templates/Darwin/Info.plist.template

```diff
@@ -35,7 +35,7 @@
 	<key>PharoImageFile</key>
 	<string>{ImageName}</string>
 	<key>PharoImageParameters</key>
-	<array>
+	<array>{Parameters}
```

Comment:
To me it looks strange that this is not in the <string> 
I do not get what this is not after the embedded